languagetool-language-modules/ga/src/main/java/org/languagetool/rules/ga/DhaNoBeirtRule.java

```diff
@@ -87,7 +84,7 @@ public String getDescription() {
       }
       if (msg != null) {
         RuleMatch match = new RuleMatch(
-          this, sentence, tokens[prevTokenIndex + 1].getStartPos(), tokens[markEnd].getEndPos(), msg, "Uimhir phearsanta");
+          this, sentence, tokens[prevTokenIndex].getStartPos(), tokens[prevTokenIndex].getEndPos(), msg, "Uimhir phearsanta");
```

Comment:
Same as before.
For the 'trí' token to be correctly underlines you need to replace previousToken by previousToken+1. This will also highlight the other numerals properly (e.g. trí et al.).
Here, since you choose to mark just one word, the replacement has to have also just one word.